package.json

```diff
@@ -18,19 +18,65 @@
             "type": "object",
             "title": "Doxygen Documentation Generator Settings",
             "properties": {
-                "doxdocgen.generic.commentStart": {
-                    "description": "Doxygen comment start indicator. Default ist /** but Qt style with /*! is also valid.",
+                "doxdocgen.generic.triggerSequence": {
+                    "description": "Doxygen comment trigger. This character sequence triggers generation of DoxyGen comments.",
                     "type": "string",
-                    "default": "/**",
-                    "enum": [
-                        "/**",
-                        "/*!"
-                    ]
+                    "default": "/**"
                 },
-                "doxdocgen.generic.generateReturnType": {
-                    "description": "Insert return type into generated documentation",
+                "doxdocgen.generic.firstLine": {
+                    "description": "The first line of the comment that gets generated. If empty it won't get generated at all.",
+                    "type": "string",
+                    "default": "/**"
+                },
+                "doxdocgen.generic.commentPrefix": {
+                    "description": "The pre fix that is used for each comment line.",
```

Comment:
prefix

Reply:
Hmm im not sure what you mean. I tried to keep the style the same as it was. You mean just name that config parameter `prefix` instead of `commentPrefix`?

Reply:
Sorry this was unclear on my part. I wanted to hint that `pre fix` should be without space `prefix`. :)

Reply:
Fixed.